src/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.csproj

```diff
@@ -51,6 +51,7 @@
     <Compile Include="File\ReadWriteAllLinesAsync.cs" />
     <Compile Include="File\ReadWriteAllBytesAsync.cs" />
     <Compile Include="File\ReadWriteAllTextAsync.cs" />
+    <Compile Include="FileStream\ReadWriteSpan.netcoreapp.cs" />
```

Comment:
> In general we don't really have a good process to make sure we enable netcoreapp tests for UAP when the changes reach there.

There are tons of tests across the repo that are guarded by netcoreapp, and presumably many of them should already be working for uap.  Sounds like we need to make a pass through.